website/src/pages/fdc3-roadmap.js

```diff
@@ -0,0 +1,160 @@
+import Layout from "@theme/Layout";
+import Container from "../components/Container"
+const React = require('react');
+
+
+const style = `
+table.roadmap img {
+  max-width: none;
+}
+
+table.roadmap td {
+  text-align: center;
+}
+
+table.roadmap th {
+  text-align: left;
+}
+
+div.centered {
+  margin: auto;
+  padding: 30px;
+}
+`
+
+export default (props) => {
+  return (
+    <Layout title="FDC3 Roadmap">
+      <Container>
+        <style>
+          {style}
+        </style>
+
+        <div class="centered">
+
+          <h1>FDC3 2.1 Roadmap</h1>
+          <p>This roadmap aims to track the standardization streams and
+            activities of the FDC3 2.1 effort.</p>
+
+          <table class="roadmap">
+            <thead>
+              <tr>
+                <th colspan="2"></th>
+                <th>Intents and Context Data</th>
+                <th>Desktop Agent Bridging</th>
+                <th>Identity</th>
```

Comment:
```suggestion
                <th colspan="2"></th>
                <th scope="col">Intents and Context Data</th>
                <th scope="col">Desktop Agent Bridging</th>
                <th scope="col">Identity</th>
```

`scope=col` and `scope=row` provide semantic information to screen readers to make it easier for blind end-users to quickly read and understand tables.